.github/workflows/pr-open.yml

```diff
@@ -304,11 +304,14 @@ jobs:
           cache: 'yarn'
 
       - name: Install dependencies
-        run: yarn --prefer-offline
+        run: yarn install --frozen-lockfile --prefer-offline
 
       - name: Build
-        run: |
-          yarn build
+        run: yarn build:production
+        env:
```

Comment:
I think you can remove the env vars. We need them only when running.

Reply:
Also on line 336 at this file, there are a lot of build args. We can get rid of them too.

Reply:
In this case, the vars are baked into the app, and they are required to be set during build operation